common/app/common/models/SelectableBean.java

```diff
@@ -2,22 +2,23 @@
 
 import io.sphere.sdk.models.Base;
 
-public class SelectableData extends Base {
+public class SelectableBean extends Base {
```

Comment:
Good point. Initially it was something generic, but now I realized it's only used for forms, so the name can be adapted.